composer.json

```diff
@@ -12,7 +13,7 @@
   ],
   "require": {
     "php": "^7.1.0",
-    "phpstan/phpstan": "^0.11.4"
+    "phpstan/phpstan": ">=0.11.6"
```

Comment:
This is unsafe, you should use `^` or `~`. Your extension can break with 0.12.

Reply:
I know, I copied the line from the extension-installer to keep both "in sync". Wasn't sure which way to go best and thought it might be good to use the same constraints as the installer.

Reply:
It's not necessary to do that. People who install the extension-installer with get enforced to have PHPStan `>=0.11.6` anyway. You should enforce the versions that this extension actually supports which is probably `^0.11.4`.